flake.nix

```diff
@@ -2,7 +2,7 @@
   description = "IOHK nix lib, packages and overlays";
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs?ref=release-22.11";
+    nixpkgs.url = "github:nixos/nixpkgs/release-25.05";
```

Comment:
Great! I keep being confused about which the right format is though.

Reply:
AFAIK, there isn't a right or wrong, I just tend to prefer the path style unless it is an ambiguous case, like a branch and a tag with the same name where the query style would disambiguate.